src/Compilers/Core/Portable/Emit/InstrumentationKind.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+    public enum InstrumentationKind
+    {
+        None = 0,
+        TestCoverage = 1,
```

Comment:
Previously, @JohnHamby had indicated that instrumentation kinds would not be an enum. I'm confused.